.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
Nice job on creating the `test.md` file. 👍but just as [required in this milestone](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), the path to this file should be placed in your `.gitignore` file. when this is done, the `test.md` file will be ignored by git which stops it from being deployed to github😉 currently, your `test.md` file is still being deployed to github😉

<img width="583" alt="ignored by git" src="https://user-images.githubusercontent.com/64643995/226821086-6a37aea8-11dd-4709-aa8a-94e6438b78bc.png">

Reply:
@Ibizugbe thank you for all the support. I went through all the problems. But due to renaming the branch I had to open a new pull request. 
this is the link to the new pull request: https://github.com/yashodhicy/Microverseday2/pull/2